internal/gcs/bridge.go

```diff
@@ -51,6 +52,9 @@ type rpc struct {
 	ch      chan struct{}
 }
 
+// todo(helsaawy): remove bridge.log entry and log based on per-request context, not
+// the context that started the bridge
```

Comment:
what bridge.log are you talking about here?

Reply:
this guy: https://github.com/microsoft/hcsshim/blob/0b8319a721ceef568157c20e5b75db88a9f6570c/internal/gcs/bridge.go#L67
It mucks things up to use that log entry for all bridge logging, instead of any entrys associated with the request context

Reply:
created a work item and removed the comment